tests/test_submdspan_static_slice.cpp

```diff
@@ -260,10 +259,10 @@ TEST(TestMdspan, SubmdspanStaticSlice_Left_iddd_FullTupleFull) {
       test_submdspan_static_slice<expected_output_mdspan_type, input_mdspan_type>(
         input_extents, stdex::full_extent, sliceSpec, stdex::full_extent);
     };
-    runTest(tuple{int(1), int{3}});
-    runTest(tuple{std::size_t(1), std::size_t(3)});
-    runTest(tuple{std::int64_t(1), std::int64_t(3)});
-    runTest(tuple{std::uint32_t(1), std::uint32_t(3)});
+    runTest(tuple<int, int>{int(1), int{3}});
```

Comment:
Oh oof, sorry about that -- thanks for fixing this!  Another option would have been to use `make_tuple`.  That would reduce code duplication, but this solution is fine for now.